<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <include file="$(find anymal_wbc)/launch/spawn.launch">
        <arg name="gui" default="false" />
        <arg name="pause" default="false" />
    </include>

    <rosparam command="load" file="$(find anymal_wbc)/config/params.yaml" ns="anymal_wbc"/>
    <node name="whole_body_controller_node" pkg="anymal_wbc" type="whole_body_controller_node" output="screen"/>
    
  <!-- RViz -->
  <node name="rviz" pkg="rviz" type="rviz"
        args="-d $(find anymal_wbc)/config/rviz/controller.rviz"
        output="screen"/>

    <node name="motion_planner_node" pkg="anymal_wbc" type="motion_planner_node" output="screen"/>
    <node name="teleop_gui_node" pkg="anymal_wbc" type="teleop_gui_node" output="screen"/>
    <node name="marker_publisher" pkg="anymal_wbc" type="marker_publisher" output="screen"/>

</launch>